cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
In the vein of what we talked about in my office, it would be great to find out if there is a simple way to include a uox_recipe.xml file in the main input.xml file.